openmls/src/group/mls_group/membership.rs

```diff
@@ -46,6 +50,94 @@ impl MlsGroup {
         self.add_members_internal(provider, signer, key_packages, true)
     }
 
+    /// Re-add members.
+    ///
+    /// If there are issues with a set of members and they are suspected to be
+    /// forked, this functions allows to remove and re-add them.
+    /// This is a convenience function that first removes, and then re-adds the
+    /// list of provided users.
+    ///
+    /// Note that this functions _does not_ enforce that the removed `members`
+    /// and new members in the `key_packages` correspond.
+    ///
+    /// For the best performance it is important to sort the key packages in the
+    /// order they appear in the tree (left to right). If `sort` is set to
+    /// `true`, this function sorts the indices internally, based on the
+    /// credential. Note that this only works if the credential doesn't change
+    /// and the credential allows uniquely identifying a leaf node.
```

Comment:
I don't understand what this is supposed to do. You can influence the order in which members are re-added, but you cannot guarantee in which leaf they will be added. The rule is always to use the left-most free leaf. If there are blanks, the members will be added in there. I also don't understand why the order matters at all.